processing/src/main/java/io/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -506,16 +506,18 @@ public void read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnCo
       )
       {
         switch (version) {
-          case UNCOMPRESSED_MULTI_VALUE:
+          case UNCOMPRESSED_MULTI_VALUE: {
```

Comment:
This change is a workaround for false positive inspection (due to a defect in the inspection itself). The inspection tracks for "false indentation", e. g.
```java
for (int i : array)
  actionOne();
  actionTwo();
```
Because I may think that both actionOne() and actionTwo() are executed in the loop, and I forgot braces.

In this case, inspection thinks that block
```java
           default:
              throw new IAE("Unsupported multi-value version[%s]", version);		              throw new IAE("Unsupported multi-value version[%s]", version);
```
should have deeper indentation, because the previous case doesn't have `break;` in the end. But if fails to see that all brances in if-else chain break the control flow (two returns and one throw).